infra/build_specified_commit.py

```diff
@@ -22,88 +22,98 @@
 import subprocess
 
 import helper
-import repo_manager
 
 
-class DockerExecutionError(Exception):
-  """An error that occurs when running a docker command."""
+class BuildData():
```

Comment:
Did linting complain about this empty paren?

Reply:
Nope.